### ISSUE TITLE
EPIC: Prepare the Toolkit page for Homepage Launch

### ISSUE THREAD

**Issue by priyanka02art** [hackforla/website#7675]
### Overview
We need to track existing issues and make new issues to deliver Toolkit page complete enough for the Homepage launch

#### Details
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

### Action Items
- [x] List all the issues related to the label
- [x] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [x] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [x] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [x] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [x] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [ ] Make the issues specified
- [ ] Link them as sub issues to this issue
- [ ] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)

### Resources/Instructions
- Parent Epic: https://github.com/hackforla/website/issues/2481

#### Used for creating this issue
- 1.01 Label name: P-Feature: Toolkit
    - 1.01.01 Issue Tab: [P-Feature: Toolkit ](https://github.com/hackforla/website/issues?q=is%3Aopen%20label%3A%22P-Feature%3A%20Toolkit%22%20)
    - 1.01.02 Project Board: [P-Feature:  Toolkit](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Toolkit%22)
- 1.02 URL of page on Hack for LA website: https://www.hackforla.org/toolkit/
- 1.03 URL of link to code search: [repo:hackforla/website NOT language:SVG NOT path:/^_data/internal/credits// TOOLKIT](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F+TOOLKIT&type=code)
- 1.04 URL of WIKI page: https://github.com/search?q=repo%3Ahackforla%2Fwebsite+TOOLKIT&type=wikis
    - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.
- 1.05 Issue for adding items to a final checklist: #8417

#### Created by this issue
- 2.01 [EXAMPLE would be a link to an issue related to delivering this page]

---
### Must Have
- TBD: Once homepage launch is ready evaluate if the toolkit page needs to be temporarily removed based on lack of viable content.

### Should Have
TBD

### Could Have
TBD

---

last updated: 2026-04-21
Highest number found: 8558

### Must Have
- [ ] #7675   
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval
      - draft: True
      - responsible: kimberlyvaldez07 
   - role: Product

- [ ] #7418 
   - why: We need to replace instances of Github and github with GitHub so that we display the company name correctly. 
   - status: Prioritized Backlog  
      - draft: False 
      - responsible: None  
   - role: Front End  

- [ ] #6161   
   - why: GitHub is a company name and should only be in lowercase (github) when in a URL, variable or directory name
   - status: In Progress (actively working)
      - draft: False 
      - responsible: daras-cu   
   - role: Back end/devOps and front end

 - [ ] #3889  
   - why: We need to make the Scalable Vector Graphics (SVGs) in the Toolkit webpage's External Resources section Web Content Accessibility Guidelines (WCAG) compliant so that we can achieve our mission of inclusivity.
   - status: Ice Box  
      - draft: False  
      - responsible: None  
   - role: Front End  

- [ ] #1848 
   - why: Ensure all website pages function correctly on mobile by identifying and fixing any mobile-specific bugs.
   - status: Ice Box  
      - draft: False  
      - responsible: None 
   - role: Design 

### Should Have
- [ ] #8558
   - why: We need to add filtering functionality to the Toolkit page so that selected filters are applied to the External Resources cards in addition to the Guide Pages cards.
   - status: Ready for Prioritization 
      - draft: False  
      - responsible: None  
   - role: Back end/devOps

- [ ] #1515 
   - why: We need to create a mobile version to ensure the code is responsive across all screen sizes.
   - status: Ice Box  
      - draft: False  
      - responsible: None  
   - role: Front End 

### Could Have
<details><summary>List of all could have issues</summary>

- #6673
   - why: This CodeQL alert is a false positive and is only being left open so that we can test a solution for eliminating false positives. Dependencies-#6548
   - status: Ice Box 
      - draft: False  
      - responsible: None  
   - role: Front end   

- #6143 
   - why: To enhance user experience, add a "List View" button on the Projects and Toolkit pages for a more concise presentation of information. 
   - status: Ice Box 
      - draft: True
      - responsible: None 
   - role: Front End   

- #3528 
   - why: Ensure consistent text elements on the Toolkit page 
   - status: Ice Box
      - draft: False
      - responsible: None
   - role: Design

- #3356
   - why: We want to add a new external resource to the website toolkit
   - status: Ice Box  
      - draft: True
      - responsible: None 
   - role: Front End 

- #918  
   - why: This information is already missing from the site with the Toolkit visible so it won't make a difference once we launch the homepage. If we do add it, it will be because we need to add the following details to the Guide on [Getting the Most Out of GitHub Issues](https://www.hackforla.org/guide-pages/github-issues.html) to enable a more comprehensive understanding for new volunteers
   - status: Ice Box 
      - draft: True
      - responsible: None 
   - role: Product
   
</details> 

### Won't have this time
<details><summary>List of all won't have issues</summary>

- #3115  

</details> 

#### Template
```
- [ ] #
   - why: 
   - status: 
      - draft: 
      - responsible: 
   - role:  
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

<details><summary>List of all issue by number</summary>

- #8558
- #7675
- #7418 
- #6673
- #6161 
- #6143
- #3889
- #3528
- #3356
- #3115
- #1848
- #1515
- #918

</details>

**Comment by priyanka02art** [comment]
- URL: https://www.hackforla.org/toolkit

- [ ] Remove the toolkit from the menu. Bring it back when its ready

**Comment by ExperimentsInHonesty** [comment]
- [ ] #7675 (this issue)
- [ ] #7668
- [ ] #7636

**Comment by priyanka02art** [comment]
# List of all issues
- [ ] #7675  
- [ ] #7668 
- [ ] #7636 
- [ ] #7418 
- [ ] #6919 
- [ ] #6673 
- [ ] #6161 
- [ ] #6143 
- [ ] #3889 
- [ ] #3528 
- [ ] #3356
- [ ] #3115
- [ ] #1848 
- [ ] #1515
- [ ] #918 


## Relevance of the issues to the Homepage launch
### Must Have
- [ ] #7675   
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval
      - draft: True
      - responsible: None 
   - role: Product
- [ ] #7418 
   - why: We need to replace instances of Github and github with GitHub so that we display the company name correctly. 
   - status: Prioritized Backlog  
      - draft: False 
      - responsible: None  
   - role: Front End    
 - [ ] #3889  
   - why: We need to make the Scalable Vector Graphics (SVGs) in the Toolkit webpage's External Resources section Web Content Accessibility Guidelines (WCAG) compliant so that we can achieve our mission of inclusivity.
   - status: Ice Box  
      - draft: False  
      - responsible: None  
   - role: Front End  
 -  [ ] #6919 (Confirm if duplicate of 7668)
   - why: In the branch feature-homepage-launch, when trying to navigate to /toolkit/ or any of the associated navigation links, the link is replaced with /toolkit/?. 
   - status: Done
      - draft: False
      - responsible: 8alpreet and daras-cu 
   - role: Back End/DevOps 
 - [ ] #7668 
   - why: URL consistency and hash navigation are broken as the hash is removed, preventing anchor link jumps. 
   - status: New Issue Approval  
      - draft: False
      - responsible: None
    - role: back end/devops

### Should Have  
- [ ] #1515 
   - why: We need to create a mobile version to ensure the code is responsive across all screen sizes.
   - status: Ice Box  
      - draft: False  
      - responsible: None  
   - role: Front End 

### Could Have
- [ ] #7636 
   - why: When a user navigates to /toolkit/ there is an unnecessary '?' appended to the end. The '?' should only be there when there is a URL query (filters in our case)
   - status: New Issue Approval  
      - draft: True  
       - responsible: None
    - role: Front End
- [ ] #6143 
   - why: To enhance user experience, add a "List View" button on the Projects and Toolkit pages for a more concise presentation of information. 
   - status: Ice Box 
      - draft: True
      - responsible: None 
   - role: Front End   
- [ ] #3528 
   - why: Ensure consistent text elements on the Toolkit page 
   - status: Ice Box
      - draft: False
      - responsible: None
   - role: Design
- [ ] #918  
   - why: We need to add the following details to the Guide on [Getting the Most Out of GitHub Issues](https://www.hackforla.org/guide-pages/github-issues.html) to enable a more comprehensive understanding for new volunteers 
   - status: Ice Box 
      - draft: True
      - responsible: None 
   - role: Product
- [ ] #6673 (not clear on what the error is)
   - why: We need to resolve the new alert [(33)](https://github.com/hackforla/website/security/code-scanning/33) and either recommend dismissal of the alert or update the code files to resolve the alert.
   - status: Ice Box 
      - draft: False  
      - responsible: None  
   - role: Front end  
- [ ] #1848 
   - why: Ensure all website pages function correctly on mobile by identifying and fixing any mobile-specific bugs.
   - status: Ice Box  
      - draft: False  
      - responsible: None 
   - role: Design

### Won't have this time
- [ ] #3115  
   - why: We need to define how users can suggest new guides on the toolkit page to maintain a steady guide queue and stay responsive to community needs.
   - status: Ice Box 
      - draft: False
      - responsible: None 
   - role: User Research 
- [ ] #3356 (not clear on what the issue) 
   - why: We want to add a new external resource to the website toolkit
   - status: Ice Box  
      - draft: True
      - responsible: None 
   - role: Front End 

#### Templates
```
- [ ] # 
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by ExperimentsInHonesty** [comment]
@aparnar1920 Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by aparnar1920** [comment]
Progress: Looking at the number of issues for the website. 
Epics
- [x] #7671 - remove from this Epic, because it's not related to the tool kit page.
- [x] #6919 - Place in Must have
   - why: This change might result in broken links.
- [x] #3356 - Move from could have to wont have this time.
   - why: As this needs to be ready to prioritize the toolkit again
- [x] #918 - Keep in the Could have revise the why
   - why: This information is already missing from the site with the Toolkit visible so it won't make a difference once we launch the homepage. If we do add it, it will be because we need to add the following details to the Guide on [Getting the Most Out of GitHub Issues](https://www.hackforla.org/guide-pages/github-issues.html) to enable a more comprehensive understanding for new volunteers

Issues with Blockers
- [x] #6673 - Move from Should have to Could Have
   - why: This CodeQL alert is a false positive and is only being left open so that we can test a solution for eliminating false positives
   - Dependencies
      - https://github.com/hackforla/website/issues/6548
- [x] #1848 - Move to Should have to Could have
   - why: This mobile responsiveness is specifically for when the published guides have sticky navs, and currently none of the published guides appearing on the toolkit have them.  See current vs future guide page layout
      - current: https://www.hackforla.org/guide-pages/survey-reporting-dashboard-guide
      - future: https://www.hackforla.org/guide-pages/how-to-set-reminders-in-slack
   - Dependencies
      - https://github.com/hackforla/website/issues/1515
      - https://github.com/hackforla/website/issues/3170
- [x] #7668 - Move to Must Have from Could Have
   - why: This change might result in broken links.
   - Dependencies:
      - https://github.com/hackforla/website/issues/7636

**Comment by aparnar1920** [comment]
Tasks to do: 
Identify changes that need to be made in order for the page to be launched,( look 1.02 Tool kit page) and leave a comment below  with any details , also look only for must have's.

**Comment by aparnar1920** [comment]
- For Issue #3348, there are two dependencies.  One is closed #3879 and the other says "When we are ready to prioritize the toolkit again" Can we move this to Prioritized Backlog from icebox?

### Items found when reviewing:  https://www.hackforla.org/toolkit

- [ ] On this card, which appears under External Resources, GitHub is current spelled Github in the title and github in body and should be changed to GitHub in both places
<details><summary>screenshot</summary>

![Image](https://github.com/user-attachments/assets/aa774420-ab66-4f4f-b04f-75b504ec6731)
</details>

**Comment by kimberlyvaldez07** [comment]
2026-08-18
Using this search
[state:open label:"P-Feature: Toolkit" is:issue](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Toolkit%22%20is%3Aissue)
This is the highest issue number found 8558 it is addressed at the top of this issue.

**Comment by kimberlyvaldez07** [comment]
- Following up on previous comment https://github.com/hackforla/website/issues/7675#issuecomment-2983977309

> ### Items found when reviewing:  https://www.hackforla.org/toolkit
> 
> - [ ] On this card, which appears under External Resources, GitHub is current spelled Github in the title and github in body and should be changed to GitHub in both places
> <details><summary>screenshot</summary>
> 
> ![Image](https://github.com/user-attachments/assets/aa774420-ab66-4f4f-b04f-75b504ec6731)
> </details> 

Left a comment on this Epic so that the issue gets made. 
- https://github.com/hackforla/website/issues/6161#issuecomment-3408307082 

Latest Update 10/28 daras-cu created the issue as follows 
- I created issue https://github.com/hackforla/website/issues/8390 to correct toolkitresources.yml and added it to the list of dependencies here
